CHANGELOG.md

```diff
@@ -1,10 +1,11 @@
 ## CHANGELOG
 
-2.X.X (In progress)
+### 2.6.0 (2020-11-26)
```

Comment:
Looks like the sample app changes are missing this one about showing granted permissions on the permissions screen: https://github.com/rakutentech/android-miniapp/commit/fb9c0a6527405f31960f6a3905ecefc28a6c2ce5

Reply:
Updated.